internal/kafka/command_replica_status_list.go

```diff
@@ -0,0 +1,146 @@
+package kafka
+
+import (
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/kafka-rest-sdk-go/kafkarestv3"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafkarest"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/utils"
+)
+
+type replicaHumanOut struct {
+	ClusterId          string `human:"Cluster"`
+	TopicName          string `human:"Topic Name"`
+	BrokerId           int32  `human:"Broker ID"`
+	PartitionId        int32  `human:"Partition ID"`
+	IsLeader           bool   `human:"Leader"`
+	IsObserver         bool   `human:"Observer"`
+	IsIsrEligible      bool   `human:"ISR Eligible"`
+	IsInIsr            bool   `human:"In ISR"`
+	IsCaughtUp         bool   `human:"Caught Up"`
+	LogStartOffset     int64  `human:"Log Start Offset"`
+	LogEndOffset       int64  `human:"Log End Offset"`
+	LastCaughtUpTimeMs string `human:"Last Caught Up Time (ms)"`
+	LastFetchTimeMs    string `human:"Last Fetch Time (ms)"`
+	LinkName           string `human:"Link Name"`
+}
+
+type replicaSerializedOut struct {
+	ClusterId          string `serialized:"cluster_id"`
+	TopicName          string `serialized:"topic_name"`
+	BrokerId           int32  `serialized:"broker_id"`
+	PartitionId        int32  `serialized:"partition_id"`
+	IsLeader           bool   `serialized:"is_leader"`
+	IsObserver         bool   `serialized:"is_observer"`
+	IsIsrEligible      bool   `serialized:"is_isr_eligible"`
+	IsInIsr            bool   `serialized:"is_in_isr"`
+	IsCaughtUp         bool   `serialized:"is_caught_up"`
+	LogStartOffset     int64  `serialized:"log_start_offset"`
+	LogEndOffset       int64  `serialized:"log_end_offset"`
+	LastCaughtUpTimeMs int64  `serialized:"last_caught_up_time_ms"`
+	LastFetchTimeMs    int64  `serialized:"last_fetch_time_ms"`
```

Comment:
After the v4 release, let's make a PR to merge these two structs into one. Certain types such as `time.Time` are always formatted in the same way based on if they're human or serialized so we can do that formatting inside the `output` package instead of outside.